### DISCUSSION TITLE
How do I use it in Windows?

### DISCUSSION THREAD

**Discussion by su-404** [teragrep/jpr_01#23] [Q&A]
How do I use it in Windows?

**Comment by kortemik** [comment]
You would need to compile the pcre2 wrapper, which resides in src/main/c, for windows and store the compiled library dll files to target/classes/win32-x86-64/. You might want to ask how to do that on stackoverflow.

We will accept a pr that compiles it with mingw cross compilation.

**Reply by su-404** [reply]
Sorry for not paying attention to the message for too long now I just saw it, can you provide the dll, I'm not familiar with the C language.
